docs/development.md

```diff
@@ -1,15 +1,19 @@
 # Development
 
+> [!IMPORTANT]
+> The `llm` package that loads and runs models is being updated to use a new [Go runner](#transition-to-go-runner): this should only impact a small set of PRs however it does change how the project is built.
+
 Install required tools:
 
-- cmake version 3.24 or higher
+- cmake version 3.24 or higher (only required for legacy C++ runner build)
 - go version 1.22 or higher
 - gcc version 11.4.0 or higher
 
+
 ### MacOS
 
 ```bash
-brew install go cmake gcc
+brew install go gcc
```

Comment:
I think we can simply point to [download Go](https://go.dev/dl/), `gcc` comes with macOS AFAIK